apps/demo-fuels/fuels.config.ts

```diff
@@ -0,0 +1,8 @@
+import { createConfig } from 'fuels';
+
+export default createConfig({
+  workspace: './backend',
+  output: './src/backend-api',
```

Comment:
I would rename `backend` for contracts or sway, just to match better the terminology for blockchains.

Reply:
I agree. Perhaps `sway` is the best option.

 1. `contracts` - too limited
 1. `programs` - too abstract
 1. `sway` - succinct
 1. `sway-programs` - descriptive but verbose

Reply:
Looking at actual examples, I think `sway-programs` is the way to go.

I'm using the `-api` suffix for the `output` directory to differentiate it from the sway `workspace`, and `sway-programs-api` seems the only reasonable option - `sway-api` would sound misleading.

```ts
import { createConfig } from 'fuels';

export default createConfig({
  workspace: './sway-programs',
  output: './src/sway-programs-api',
  ...
});
```